pyaedt/edb_core/edb_data/padstacks_data.py

```diff
@@ -1504,6 +1499,7 @@ def metal_volume(self):
     @property
     def pin_number(self):
         """Get pin number."""
+        warnings.warn("`pin_number` is deprecated. Use `name` property instead.", DeprecationWarning)
```

Comment:
I guess you want to to say use `number`instead ?

Reply:
agreed there is no notion of pin number, the initial property pin_number is not correct. Let change the deprecation to `name` as suggested